api/users/index.js

```diff
@@ -0,0 +1,44 @@
+const router = require("express").Router();
+const createUser = require("../../helpers/Users/createUser");
+const { v4 } = require("uuid");
+const UpdateUser = require("../../helpers/Users/updateUsers");
+const loginMiddleware = require("../../middleware/checkLoggedIn");
+
+//register
+router.post("/", async(req, res) =>{
+  const { email, password1, password2, type } = req.body;
+  if(!email || !password1 || !password2 || !type) {
+    return res.sendStatus(400);
+  }
+  if(password1 != password2) {
+    // flash error message later
```

Comment:
remove this comment

Reply:
c4f363c71f87c14edcda4ccab29f044dfc6d3963